test/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "bedrock-web-store-test",
+  "version": "0.0.1-0",
+  "description": "Bedrock web store test",
+  "scripts": {
```

Comment:
add `private` to test package files.  Prevents accidental NPM publishing from the wrong folder.